### DISCUSSION TITLE
[@types/bitcore-lib] Incorrect function signature for toAddress

### DISCUSSION THREAD

**Discussion by krtk6160** [DefinitelyTyped/DefinitelyTyped#55690] [Issues with a @types package]
The `toAddress` function in [bitcore-lib](https://github.com/bitpay/bitcore/blob/v8.0.0/packages/bitcore-lib/lib/script/script.js#L1056) can optionally accept a `network` param, which makes it possible to parse testnet/regtest transactions and scripts and return correct addresses.

For now, a ts-ignore comment needs to be used when passing the network param to `toAddress`.
The function signature in [@types/bitcore-lib](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/bitcore-lib/index.d.ts#L252) needs to be updated to fix the problem.

[a comment by an automated account is not shown]

**Comment by lautarodragan** [comment]
@krtk6160 I'm no longer using bitcore-lib, I dropped it since it seemed pretty much abandoned and using a bitcoin core node directly wound up being as easy tbh.

Anyways, feel free to update the type definitions for bitcore-lib and ping me in the PR.
